workspace sir vivor/commands.js

```diff
@@ -2367,17 +2367,24 @@ let commands = {
 			let points = dd.getPoints(sorted[i]);
 			let bgcolor = dd.getBgColor(sorted[i]);
 			let textcolor = dd.getTextColor(sorted[i]);
+			let displaypoints = dd.getDisplayPoints(sorted[i]);
 			if (points === 0) continue;
 			let h = hostcount.count[toId(cur)] ? hostcount.count[toId(cur)] : 0;
 			let n = gamecount.count[toId(cur)];
 			let e = eventcount.count[toId(cur)] ? eventcount.count[toId(cur)] : 0;
 			if (!n) n = "Error";
 
 			points = Math.floor(50 * (points/n) * ((n*n)/(n*n+50)) + ((h*h+300)/300)) + e;
+			
+			if (displaypoints >= points) {
+				points = displaypoints;	
+			} else {
+				dd.updateDisplayPoints(cur, points);
+			}
 
 			res.push([
 				cur,
-				points,
+				displaypoints,
```

Comment:
Given that you set `points = displaypoints` above, wouldn't just `points` here work?